encointer/encointer-parachain
# Add try-runtime with storage migrations of the encointer-pallets

Closes https://github.com/encointer/encointer-parachain/issues/176 in order to test https://github.com/encointer/encointer-parachain/issues/175

In order to test this, do the following:

* Check out https://github.com/encointer/pallets/tree/migrations-for-weight-v2
* Check out https://github.com/encointer/encointer-node/tree/try-runtime and run `cargo build --release --features=try-runtime`
* Check out https://github.com/encointer/encointer-parachain/tree/try-runtime and run `cargo build --release --features=try-runtime`
* Then in the node run:
```
cargo run --bin encointer-node-notee --release --features=try-runtime try-runtime --runtime ../encointer-parachain/target/release/wbuild/encointer-runtime/encointer_runtime.wasm on-runtime-upgrade --checks=all live --uri wss://kusama.api.encointer.org:443
```

This will simulate the migration including pre- and postmigration checks with the chain data from Kusama.
